### ISSUE TITLE
Tough As Nails Compatibility Issue

### ISSUE THREAD

**Issue by Megafoan** [Flaxbeard/Cyberware#139]
Cyberware's cyberware that is meant to interact with Tough As Nails' temperature (the sweat glands and the blubber) does not appear to actually modify the temperature gain or loss to my knowledge, based on watching the /tan tempinfo. I am running Cyberware 0.2.10 and Tough As Nails 2.1.0.51. For instance, I have the sweat glands installed but still burn up in the Nether.

**Comment by VallenFrostweaver** [comment]
I'm actually experiencing client crashing on a multiplayer server for both of those items running Raxpack 2 which has TAN in it.

**Comment by Megafoan** [comment]
Yeah, I'm using Raxpack 2 as well. Single player isn't experiencing a client crash, though. But what's happening to you could mean that what is happening to me isn't normal.

**Comment by VallenFrostweaver** [comment]
Informatio about this error version: Minecraft 1.11.2 
Cyberware version: alpha-0.2.10

**Comment by 23Link89** [comment]
Cyberware doesn't support 1.11.2 yet. You're on your own until there's official support.

**Comment by gajbooks** [comment]
I also have this issue, but only for the Supercooled Sweat Glands. The blubber works fine. I'm using Minecraft 1.10.2, Tough As Nails 1.1.1.66, Cyberware beta 0.2.10.
